apps/system-e2e/src/playwright.config.ts

```diff
@@ -42,79 +34,15 @@ const config: PlaywrightTestConfig = {
         ],
         ['html', { open: 'never' }],
       ]
-    : 'html',
+    : 'line',
```

Comment:
I think at this early stage of the adoption of the testing, it might be better to opt for the more verbose 'html' setup to be more helpful troubleshoot issues.